llm-request-plz.el

```diff
@@ -152,7 +152,7 @@ the buffer is turned into JSON and passed to ON-SUCCESS."
             (when on-success-raw
               (funcall on-success-raw response))
             (when on-success
-              (funcall on-success (json-read-from-string response))))
+              (funcall on-success (plz-response-body response))))
```

Comment:
This won't work for async requests, though (since in that case it just returns a string).  I checked in a fix for this.  Thank you for catching the issue!